readme.md

```diff
@@ -21,9 +21,9 @@ before using this repository. **If you do not, `kobo-install` will not be able t
 
 ## Versions
 
-Branch `master` is the recommended branch to use `kobo-install` on your production environment. Just run `git checkout master` before your first run.
+Release branches `release/*` (e.g. `release/2.024.36`) are the recommended branches to use with `kobo-install` on your production environment. Run `git switch "$(git branch -rl 'origin/release/*' | tail -1 | sed -r 's/^.*n\///')"` to switch to the latest release branch.
```

Comment:
:sunglasses: nice to include this! 

suggestion:
- simpler sed command: `sed -r 's/^(origin\/)//'`
- emphasize to run this in `kpi` repository (not kobo-install)

Reply:
- Simpler sed won't work, because there are 2 leading spaces.
- good point, will rephrase.